x-pack/libbeat/common/aws/credentials.go

```diff
@@ -94,8 +100,13 @@ func getSharedCredentialProfile(config ConfigAWS) (awssdk.Config, error) {
 		return awsConfig, errors.Wrap(err, "external.LoadDefaultAWSConfig failed with shared credential profile given")
 	}
 
-	// Set default region to make initial aws api call
-	awsConfig.Region = "us-east-1"
+	if awsConfig.Region == "" {
```

Comment:
This reads that the region in the loaded AWS configuration is taken over the region passed into the beat, is that expected behavior? I would expect that if beats configuration had an explicit region it would override the config from AWS? Unless that has an unknown side-effect?